<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeForm</class>
 <widget class="QWidget" name="VolumeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>35</width>
    <height>161</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>35</width>
    <height>161</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>35</width>
    <height>161</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image:url(:/image-assets/preview-volume-background.png);
background-color:#F7F7F7;
border:0px</string>
  </property>
  <widget class="QPushButton" name="btnVolumeAdd">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-3</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/image-assets/preview-volume-up.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVolumeSub">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>131</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/image-assets/preview-volume-down.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="sliderPreviewVolume">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>30</y>
     <width>37</width>
     <height>102</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>35</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url();
background-color:#F7F7F7;
border:0px</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
